### ISSUE TITLE
Not displaying

### ISSUE THREAD

**Issue by risen372** [eerussianguy/BlazeMap#17]
Blaze map world map is blank with icons on the left. Mini map works fine.

**Comment by BardSymphony** [comment]
Yes I have the exact same problem, I don't understand why. Shift+B open the mini-map and display it correctly, but when I press B it's just grey screen with the icon on the left.

**Comment by aionophe** [comment]
Same issue. I noticed that world map stops displaying when I enable FTBChunks minimap.
